src/main/java/org/breedinginsight/daos/ProgramUserDAO.java

```diff
@@ -194,4 +194,17 @@ public List<ProgramUser> parseRecords(Result<Record> records, BiUserTable create
 
         return resultProgramsUsers;
     }
+
+    public List<ProgramUser> getProgramUsersByRole(UUID programId, UUID roleId) {
+        BiUserTable createdByUser = BI_USER.as("createdByUser");
+        BiUserTable updatedByUser = BI_USER.as("updatedByUser");
+
+        Result<Record> records = getProgramUsersQuery(createdByUser, updatedByUser)
+                .where(PROGRAM.ACTIVE.eq(true))
+                .and(PROGRAM.ID.eq(programId))
+                .and(ROLE.ID.eq(roleId))
```

Comment:
I think this should also have `.and(PROGRAM_USER_ROLE.ACTIVE.eq(true)` like in the `existsAndActive` method to make sure that the program user is still active in the program.

Reply:
Done.